docker/vpp-vswitch/vpp/Dockerfile

```diff
@@ -1,18 +1,25 @@
-FROM ubuntu:16.04
+FROM ubuntu:18.04
 
 RUN apt-get update \
  && apt-get install -y --no-install-recommends \
-    sudo wget git gdb nano python \
-    iproute2 iputils-ping inetutils-traceroute libapr1 supervisor \
-    telnet netcat software-properties-common \
-    make autoconf automake libtool curl unzip \
-    ethtool\
+    # general tools
+    sudo wget git ca-certificates \
+    # build tools
+    make \
+    # vpp-agent & vpp process manger
+    python supervisor \
+    # required for disabling TCP checksum offload in containers
+    ethtool \
```

Comment:
Btw. checksum offloading is now disabled by vpp-agent via syscall, but I would keep ethtool installed anyway, just for debugging purposes.